app/utils.py

```diff
@@ -132,3 +132,15 @@ def hilite(message):
     ansi_green = "\033[32m"
     ansi_reset = "\033[0m"
     return f"{ansi_green}{message}{ansi_reset}"
+
+
+def aware_utcnow():
+    return datetime.now(timezone.utc)
```

Comment:
This method would be for the future and what `utc_now()` would start calling under the hood when getting back to being timezone aware?